cnd/Cargo.toml

```diff
@@ -11,7 +11,7 @@ anyhow = "1"
 async-trait = "0.1"
 base64 = "0.12.0"
 bigdecimal = "0.1.0"
-bitcoin = { version = "0.23", features = ["use-serde"] }
+bitcoin = { version = "0.23", features = ["use-serde", "rand"] }
```

Comment:
We only use the `rand` feature in tests so you could also just add a `bitcoin` dependency with the `rand` feature in `dev-dependencies`.